src/vm/hints/hint_utils.rs

```diff
@@ -162,13 +162,14 @@ pub fn is_nn_out_of_range(
                         return Err(VirtualMachineError::NoRangeCheckBuiltin);
                     };
                     //Main logic (assert a is not negative and within the expected range)
-                    let mut value = bigint!(1);
-                    if (-a.clone() - bigint!(1)).mod_floor(&vm.prime) >= bigint!(0)
+                    let value = if (-a.clone() - bigint!(1)).mod_floor(&vm.prime) >= bigint!(0)
```

Comment:
`.mod_floor` is always non-negative. There's also no need for cloning `a`.
```suggestion
                    let value = if (-a - &bigint!(1)).mod_floor(&vm.prime)
```

Reply:
Done!